File: VC/1.0.4/web/project/Care4Toolbox/src/main/java/com/care4u/toolbox/ToolboxDto.java
package com.care4u.toolbox;

import com.care4u.hr.membership.MembershipDto;

import lombok.Builder;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.ToString;

@Getter
@NoArgsConstructor
@ToString

public class ToolboxDto {
	
	private long id;
	
	private String name;
	
	private MembershipDto managerDto;
	
	private boolean systemOperability;
	
	@Builder
	public ToolboxDto(long id, String name, MembershipDto managerDto, boolean systemOperability) {
		this.id = id;
		this.name = name;
		this.managerDto = managerDto;
		this.systemOperability = systemOperability;
	}
	
	public ToolboxDto(Toolbox toolbox) {
		this.id = toolbox.getId();
		this.name = toolbox.getName();
		this.managerDto = new MembershipDto(toolbox.getManager());
		this.systemOperability = toolbox.isSystemOperability();
	}
		
}

File: App/BTClient/app/src/main/java/com/mrsmart/standard/toolbox/ToolboxDto.kt
package com.mrsmart.standard.toolbox

import com.mrsmart.standard.membership.MembershipDto

data class ToolboxDto(
    val id: Long,
    val name: String,
    //val managerDto: MembershipDto, //얘는 나중에 그래도 어느 실 manager인지 판단해서 자동 로그인할 수 있게 하면 좋을 것 같은데
    //val systemOperability: Boolean
)